src/components/CurriculumComponents/CurriculumVisualiser/CurriculumVisualiser.tsx

```diff
@@ -294,13 +285,13 @@ const CurriculumVisualiser: FC<CurriculumVisualiserProps> = ({
                 >
                   Year {year}
                 </OakHeading>
-                {disciplines?.length > 1 && (
-                  <Box role="group" aria-label="Disciplines">
-                    {disciplines.map((discipline, index) => {
-                      const isSelected = isSelectedDiscipline(
+                {subjectCategories?.length > 1 && (
+                  <Box role="group" aria-label="Subject category">
```

Comment:
For consistency with Pupil squad we should go with an ARIA label of 'Categories' - note also the pluralisation as we're naming the group not the items within it.